app.js

```diff
@@ -31,11 +41,13 @@ app.use(express.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 // Public routes which don't require a TOKEN
-app.post("/api/v1/auth", getAuthToken)
+app.post("/api/v1/auth/api", getAuthToken)
+app.get("/api/v1/auth/google", googleAuthRedirect);
+app.get("/api/v1/auth/google/callback", googleAuthCallback);
```

Comment:
our endpoint paths should be defined in our paths file.

Reply:
New router with endpoint created